oak-core/src/main/java/org/apache/jackrabbit/oak/query/QueryEngineSettingsService.java

```diff
@@ -82,6 +92,7 @@
     static final String QUERY_FAIL_TRAVERSAL = "queryFailTraversal";
     
     static final String QUERY_FAST_QUERY_SIZE = "fastQuerySize";
+    static final String DISABLED_STRICT_PATH_RESTRICTION = "DISABLE";
```

Comment:
Maybe better use StrictPathRestriction.DISABLE? I guess you can import StrictPathRestriction here right?

Reply:
This value is being used in interface above with this as default value and default value needs to be a constant so I had to use a String instead of StrictPathRestriction.DISABLE

Reply:
OK, I didn't know... It's fine then!